### ISSUE TITLE
Re-introduce non-forked containers

### ISSUE THREAD

**Issue by earldouglas** [earldouglas/xsbt-web-plugin#221]
This is a placeholder to accumulate +1's from the community, gather the use cases, and build up a compelling reason to bring back non-forked containers in xsbt-web-plugin.

If you would like to see this feature re-introduced, please leave a comment describing your use case.  Hopefully we can address them all with the forked JVM behavior, but if not, let's make it clear that it's insufficient.

**Comment by olegstepura** [comment]
Just to mention is here as well:

On dev environment now there is an issue that if you close the terminal, the jetty instance is not closed and is still running. Trying to start the project second time causes `BindException`

```
2015-02-02 19:08:02.014:WARN:oejuc.AbstractLifeCycle:main: FAILED ServerConnector@75c56eb9{HTTP/1.1}{0.0.0.0:8000}: java.net.BindException: Address already in use: bind
java.net.BindException: Address already in use: bind
```

You have to go to task manager and kill the process manually.

**Comment by DavidPerezIngeniero** [comment]
What are the advantages of non-forked containers?  
More speed to launch? 
Ability to reload changes?

**Comment by earldouglas** [comment]
There's a minor speed advantage, since it takes time to spin up a cold JVM in a new process.

**Comment by earldouglas** [comment]
Since the underlying use cases all have to do with rapid turnaround, I'm closing this issue in favor of #236.

**Comment by earldouglas** [comment]
Check out [quickstart mode][1], now available in xsbt-web-plugin 2.3.0.

[1]: https://github.com/earldouglas/xsbt-web-plugin/blob/2.3.0/docs/2.3.md#quickstart-mode
